src/components/AddResourceModal/AddBathroom.jsx

```diff
@@ -113,7 +113,7 @@ function AddBathroom({
     }
   ];
 
-  const userLocation = useSelector(state => state.userLocation);
+  const userLocation = useSelector(state => state.filterMarkers.userLocation);
```

Comment:
I was thinking it might make sense split `filterMarkers.js` into different files and rename appropriately to reflect the contents better. In particular, the action `SET_TOOLBAR_MODAL` doesn't seem to fit into the "filterMarkers" category, I think. Any thoughts? 

Since this PR will require changing the approach to referencing the `filterMarkers` reducer, it seemed like a good time to bring this up, though it may make sense to address in a separate follow-up issue if we decide to do that.

Reply:
It makes sense, but it will be a significant shift for this, as we need to map all state fields into new slices, and touching this part of the code can be very fragile. 
I think it's worth having a refactor ticket that will be dedicated to this body of work

Reply:
I drafted an issue to follow up on this thread here: https://github.com/phlask/phlask-map/issues/432